pymc/distributions/discrete.py

```diff
@@ -1761,16 +1761,16 @@ class _OrderedProbit(Categorical):
     rv_op = categorical
 
     @classmethod
-    def dist(cls, eta, cutpoints, *args, **kwargs):
+    def dist(cls, eta, cutpoints, sigma, *args, **kwargs):
```

Comment:
```suggestion
    def dist(cls, eta, cutpoints, sigma=1, *args, **kwargs):
```
To make it optional

Reply:
Yep, also going to update that and the tests.